sourmash_lib/__init__.py

```diff
@@ -29,7 +31,8 @@ class Estimators(object):
     """
 
     def __init__(self, n=None, ksize=None, protein=False,
-                 with_cardinality=False, track_abundance=False):
+                 with_cardinality=False, track_abundance=False,
+                 max_hash=0, seed=DEFAULT_SEED):
         "Create a new MinHash estimator with size n and k-mer size ksize."
```

Comment:
done! and thanks - I'd missed that `protein` was different from `is_protein` in this code. ac43885